Dockerfile

```diff
@@ -49,11 +71,17 @@ RUN pip install --no-cache-dir --upgrade pshtt==0.6.6
 
 ###
 # Install domain-scan
-###
-RUN git clone https://github.com/18F/domain-scan \
-    ${HOME}/domain-scan/
+#
+# The SHELL command is used to ensure that if either the curl call or
+# the tar call fail then the image build fails. Source:
+# https://github.com/docker/docs/blob/main/develop/develop-images/dockerfile_best-practices.md#using-pipes
```

Comment:
Sorry since I'm the one who suggested the URL that was here in the first place. I just figured this is better than a GitHub repository link.

```suggestion
# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#using-pipes
```